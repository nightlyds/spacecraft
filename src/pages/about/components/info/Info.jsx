import React from "react";
import Translate from "react-translate-component";
import GitHub from "../../images/github.svg";
import Instagram from "../../images/instagram.svg";

const Info = () => {
  return (
    <div className="about-info">
      <div className="about-info-title-box">
        <Translate
          component="h2"
          content="infoTitle"
          className="about-info-title"
        />
      </div>
      <div className="about-info-description">
        <Translate
          component="p"
          content="infoDescription"
          className="about-info-description"
        />
      </div>
      <div className="about-info-social">
        <div className="about-info-social-box">
          <a href="https://github.com/nightlyds/">
            <img src={GitHub} className="about-info-social-icon" alt="GitHub" />
          </a>
        </div>
        <div className="about-info-social-box">
          <a href="https://www.instagram.com/safonovdaniel/">
            <img
              src={Instagram}
              className="about-info-social-icon about-info-social-icon-last"
              alt="Instagram"
            />
          </a>
        </div>
      </div>
    </div>
  );
};

export default Info;
