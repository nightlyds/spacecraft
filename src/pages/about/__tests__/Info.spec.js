import React from 'react';
import Info from '../components/info/Info.jsx'
import shallow from 'enzyme/build/shallow';

const setUp = (props) => shallow(<Info {...props} />)

describe("About component tests", () => {
    let component;

    beforeEach(() => {
        component = setUp();
    })

    it("Snapshot", () => {
        expect(component).toMatchSnapshot();
    })
})