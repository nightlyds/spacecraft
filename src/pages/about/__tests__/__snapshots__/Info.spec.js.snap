// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component tests Snapshot 1`] = `
<div
  className="about-info"
>
  <div
    className="about-info-title-box"
  >
    <Translate
      className="about-info-title"
      component="h2"
      content="infoTitle"
    />
  </div>
  <div
    className="about-info-description"
  >
    <Translate
      className="about-info-description"
      component="p"
      content="infoDescription"
    />
  </div>
  <div
    className="about-info-social"
  >
    <div
      className="about-info-social-box"
    >
      <a
        href="https://github.com/nightlyds/"
      >
        <img
          alt="GitHub"
          className="about-info-social-icon"
          src="test-file-stub"
        />
      </a>
    </div>
    <div
      className="about-info-social-box"
    >
      <a
        href="https://www.instagram.com/safonovdaniel/"
      >
        <img
          alt="Instagram"
          className="about-info-social-icon about-info-social-icon-last"
          src="test-file-stub"
        />
      </a>
    </div>
  </div>
</div>
`;
